src/Akeneo/Connectivity/Connection/back/Infrastructure/Symfony/Resources/translations/jsmessages.en_US.yml

```
@@ -222,7 +233,7 @@ akeneo_connectivity.connection:
             not_auditable:
                 title: Unmonitored connection
                 description:
-                    1: To track errors on this connection, 
+                    1: To track errors on this connection,
```

To avoid an unnecessary update on Crowdin
```suggestion
                    1: To track errors on this connection, 
```